helpfunctions/utils.py

```diff
@@ -207,17 +207,21 @@ def getLeaderboardPageBy(self, page, sortBy):
                 # When user is not in guild.
                 nick = "Not on guild"
                 name = f"ID: {userID}"
+            # Check length of nick + name
+            nick = nick if len(nick) <= 12 else nick[:10] + "..."
+            name = name if len(name) <= 22 else name[:20] + "..."

```

Comment:
When the length of the nick is `13` then it's three last characters will be replaced, but it is still the same leght. Thus, it would not be needed to replace them with "...".
See:
```
11 aaaaaaaaaaa
12 aaaaaaaaaaaa
13 aaaaaaaaaa...
14 aaaaaaaaaa...
```